Formula/ford.rb

```diff
@@ -18,7 +18,7 @@ class Ford < Formula
   end
 
   depends_on "graphviz"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
```

Comment:
Did you do `brew update-python-resources` after changing the dependency? They're different for different versions sometimes.